packages/db/src/scripts/helpers/parse-lexicon.ts

```diff
@@ -1,36 +1,44 @@
 import { createReadStream } from 'fs';
 import { createInterface } from 'readline';
 
-/**
- * This is used to parse our lexicon files, so that they can be inserted into
- * the DB.
- */
-export const parseLexicon = async (
-  filename: string,
-  keys: string[]
-): Promise<Record<string, Record<string, string>>> => {
-  const input = createReadStream(filename);
-  const reader = createInterface({ input });
-  const parsed: Record<string, Record<string, string>> = {};
+export const parseBdb = async (filename: string) => {
+  const parsed: Record<string, string> = {};
   let currentId = '';
-  let currentData: Record<string, string> = {};
-  for await (const line of reader) {
+  await lineByLine(filename, (line) => {
     const indicator = line[0];
     const rest = line.substring(1);
-    if (indicator == '$') {
-      if (currentId) {
-        // Record the last data before overwriting the temporary variables.
-        parsed[currentId] = currentData;
-      }
-      currentData = {};
-    } else if (indicator == '@') {
+    if (indicator == '@') {
       const [key, value] = rest.split('=\t', 2);
-      if (keys.includes(key)) {
-        currentData[key] = toMarkDown(value);
+      if (key == 'BdbMedDef') {
+        parsed[currentId] = toMarkDown(value);
       } else if (key == 'StrNo') {
         currentId = value;
       }
     }
+  });
+  return parsed;
+};
+
+export const parseLsj = async (filename: string) => {
+  const parsed: Record<string, string> = {};
+  await lineByLine(filename, (line) => {
+    const split = line.split('\t');
+    const id = split[1].split('=')[0].trim();
```

Comment:
I think we are going to be using the eStrongs number across the board. In Hebrew that aligns with the BDB. #248 should give us a datasource that aligns with eStrongs so we'll be able to update each word to match these LSJ entries. Can you modify this to use that as the strongs?

Reply:
I was using the `dStrong` column so that we could differentiate between these entries.

Reply:
For now, let's take the first entry

Reply:
Done. Also improved parsing of bold/italics indicators.